--- a/debian/control
+++ b/debian/control
@@ -17,7 +17,7 @@ Build-Depends: debhelper (>= 13),
  python3-scapy
 Homepage: https://github.com/solemnwarning/powernap
 
-Package: powernap
+Package: powernap-git
 Architecture: all
 Depends:
  ${misc:Depends},
@@ -26,7 +26,7 @@ Depends:
  pm-utils,
  python3-pytimeparse
 Recommends: ethtool
-Enhances: powerwake
+Enhances: powerwake-git
 Description: reduce the power consumption of a system when inactive
  PowerNap watches a series of configurable monitors.  When no activity
  has occurred on any of these monitors for some specified time, PowerNap
@@ -49,17 +49,17 @@ Description: reduce the power consumption of a system when inactive
  .
  See /etc/powernap/config for all configurable options and defaults.
 
-Package: powerwake
+Package: powerwake-git
 Architecture: all
 Depends:
  ${misc:Depends},
  ${python3:Depends},
- powerwake-common (= ${binary:Version}),
+ powerwake-common-git (= ${binary:Version}),
  python3-dnspython,
  python3-netifaces,
  python3-pyipmi,
  debconf (>= 0.5) | debconf-2.0
-Enhances: powernap
+Enhances: powernap-git
 Description: remotely wake a napping system
  PowerWake is a generic mechanism for remotely waking systems.  It is
  intended to support wake-on-lan, ipmi, and other remote waking mechanisms.
@@ -67,11 +67,11 @@ Description: remotely wake a napping system
  some handy caching of MAC addresses, such that systems can be awakened by
  hostname or ip address, in addition to MAC address.
 
-Package: powerwaked
+Package: powerwaked-git
 Architecture: all
 Depends: ${misc:Depends}, ${python3:Depends}, debconf (>= 0.5) | debconf-2.0,
- powerwake-common (= ${binary:Version})
-Enhances: powerwake
+ powerwake-common-git (= ${binary:Version})
+Enhances: powerwake-git
 Description: remotely wake a napping system - Server Daemon
  PowerWake Server is a mechanism to remotely monitor and wake up systems
  based on manual intervention, or automatic intervention based on a set of
@@ -84,7 +84,7 @@ Description: remotely wake a napping system - Server Daemon
  When activity is received, the monitors will automatically try to wake up
  if it is in powersave or offline mode.
 
-Package: powerwake-common
+Package: powerwake-common-git
 Architecture: all
 Depends: ${misc:Depends}, ${python3:Depends}, debconf (>= 0.5) | debconf-2.0, python3-scapy
 Description: common library files required by powerwake
